src/Kernel32.Desktop/Kernel32+InputEventTypeFlag.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="InputEventTypeFlag"/> nested type.
+    /// </content>
+    public partial class Kernel32
+    {
+        /// <summary>
+        /// A handle to the type of input event and the event record stored in the <see cref="INPUT_RECORD.Event"/> member.
+        /// </summary>
+        public enum InputEventTypeFlag
```

Comment:
Are you missing a `[Flags]` attribute here (and if so, please pluralize the type name: perhaps `InputEventTypes`)